src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/BasicExpressionCompilerTest.vbproj

```diff
@@ -99,5 +99,8 @@
   <ItemGroup>
     <Folder Include="My Project\" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
```

Comment:
CC @davkean 

Part of our new SDK effort is keeping csproj files as minimal as possible.  We should be very skeptical of any service that is auto-adding unneeded entries into the project file.